core/modules/file/file.install

```diff
@@ -648,3 +648,91 @@ function file_update_1000() {
     $config->save();
   }
 }
+
+/**
+ * Add a Manage file link to the admin files view.
+ */
+function file_update_1001() {
+  $config = config('views.view.file_admin');
+
+  // Get all the fields.
+  $fields = $config->get('display.default.display_options.fields');
+
+  // If the file field is not already there...
+  if (!array_key_exists('edit', $fields)) {
+
+    // Remove the delete and drop-button fields.
+    $delete = $fields['delete'];
+    unset($fields['delete']);
+    $dropbutton = $fields['dropbutton'];
+    unset($fields['dropbutton']);
+
+    // Add the Manage file field.
+    $fields['edit'] = array(
+      'id' => 'edit',
+      'table' => 'file_managed',
+      'field' => 'edit',
+      'relationship' => 'none',
+      'group_type' => 'group',
+      'ui_name' => '',
+      'label' => 'Manage link',
+      'exclude' => TRUE,
+      'alter' => array(
+        'alter_text' => 0,
+        'text' => '',
+        'make_link' => 0,
+        'path' => '',
+        'absolute' => 0,
+        'external' => FALSE,
+        'replace_spaces' => 0,
+        'path_case' => 'none',
+        'trim_whitespace' => 0,
+        'alt' => '',
+        'rel' => '',
+        'link_class' => '',
+        'prefix' => '',
+        'suffix' => '',
+        'target' => '',
+        'nl2br' => 0,
+        'max_length' => '',
+        'word_boundary' => TRUE,
+        'ellipsis' => TRUE,
+        'more_link' => 0,
+        'more_link_text' => '',
+        'more_link_path' => '',
+        'strip_tags' => 0,
+        'trim' => 0,
+        'preserve_tags' => '',
+        'html' => 0,
+      ),
```

Comment:
I usually exclude the `alter` array if all the options are the default, then they will be repopulated by Views on the loading of the view.

Reply:
nice, done.